Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -244,6 +244,14 @@ class WebProcessPool final
 
     void handleMemoryPressureWarning(Critical);
 
+#if PLATFORM(COCOA)
+    void screenPropertiesChanged();
+#endif
+
+#if PLATFORM(MAC)
+    void displayPropertiesChanged(const WebCore::ScreenProperties&, WebCore::PlatformDisplayID, CGDisplayChangeSummaryFlags);
```

Comment:
CGDisplayChangeSummaryFlags should really be wrapped in a platform independent type, and then this whole thing could lose the #ifdefs

Reply:
I considered doing that but all the downstream code is #ifdeffed for macOS, and translating between CGDisplayChangeSummaryFlags and a WebKit type seemed like too much complexity for the benefit.